tests/integration/api_v3/rpc/posValidators.test.js

```diff
@@ -147,7 +148,7 @@ describe('pos/validators API', () => {
 			expect(response).toMap(jsonRpcEnvelopeSchema);
 			const { result } = response;
 			expect(result).toMap(validatorsResponseSchema);
-			expect(result.data.length).toBeGreaterThanOrEqual(1);
+			expect(result.data.length).toBeGreaterThanOrEqual(0);
```

Comment:
Why did we set it to be greater than equal to `0` in all the testcases?

Reply:
Its set to 0 since we request with offset=1